PeerioTechnologies/peerio-mobile
# Feat tm save ak text

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/14341/mobile-android-add-mixpanel-tracker-for-downloading-txt-version-of-account-key
#### Testing instructions  
Using and Android device which does not have a PDF viewer:
Verify that the properties sent by clicking "save PDF" during sign up backup account key step match the properties described in the CH story.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
